tests/utils.py

```diff
@@ -36,14 +37,25 @@ def get_ray_servlet_and_obj_store(env_name):
     return test_servlet, test_obj_store
 
 
-def get_ray_cluster_servlet(cluster_config=None):
+def get_ray_cluster_servlet(
+    cluster_config: Optional[Dict] = None, name: Optional[str] = None
+):
     """Helper method for getting base cluster servlet"""
-    cluster_servlet = get_cluster_servlet(create_if_not_exists=True)
 
-    if cluster_config:
-        ObjStore.call_actor_method(
-            cluster_servlet, "aset_cluster_config", cluster_config
-        )
+    import logging
+
+    import ray
+
+    # need to initialize ray so the cluster servlet will be initialized.
+    ray.init(
```

Comment:
confused how this was working before this change?

Reply:
don't think we used `get_ray_cluster_servlet()` before...
It was called only in the `test_cluster_servlet` fixture, which isn't called anywhere in the tests. Think we can delete this fixture for now.